### PR TITLE
Cpr to validate links

### PR DESCRIPTION
**Pull request by neetuFT**
deploying new instance of CPR to coco

### CODE REVIEW

**Comment by yolandeleungFT** [comment]
+1

**Comment by yolandeleungFT** [comment]
+1

File: content-public-read-sidekick@.service [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[Unit]
```
Comment (catalin-ilea) [inline content-public-read-sidekick@.service]: Can you please use the new sk pattern? This avoids the issue with missing ports in vulcan backends:
https://github.com/Financial-Times/up-service-files/blob/master/document-store-api-sidekick%40.service

File: content-public-read@.service [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# work correctly.
+KillMode=none
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill %p-%i > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm %p-%i > /dev/null 2>&1'
+ExecStartPre=/bin/bash -c 'docker pull up-registry.ft.com/coco/content-public-read:$DOCKER_APP_VERSION'
+
+# Setting the proper API_HOST parameter according to the current environment
+ExecStart=/bin/sh -c "\
+ENV_TAG=$(etcdctl get /ft/config/environment_tag) ; \
+if [ $ENV_TAG = "prod-uk" ] || [ $ENV_TAG = "prod-us" ]; \
```
Comment (catalin-ilea) [inline content-public-read@.service]: This seems a bit fragile and error prone if we ever decide to add more prod clusters. Maybe pass in API_HOST as a param in the provisioner and keep the service file simple and generic?

Comment (catalin-ilea) [inline content-public-read@.service]: Also unescaped double quotes

File: services.yaml [hunk 3]
```diff
@@ -89,7 +89,12 @@ services:
 - name: content-public-read-preview-sidekick@.service
   count: 2
 - name: content-public-read-preview@.service
-  version: v40
+  version: v43
+  count: 2
+- name: content-public-read-sidekick@.service
+  count: 2
+- name: content-public-read@.service
```
Comment (catalin-ilea) [inline services.yaml]: This is a read service...should have sequential deployment enabled.

**Pull request merged**